endpoints/api_request.py

```diff
@@ -50,6 +50,9 @@ def __init__(self, environ, base_paths=None):
     self.server = environ['SERVER_NAME']
     self.port = environ['SERVER_PORT']
     self.path = environ['PATH_INFO']
+    self.request_uri = environ.get('REQUEST_URI')
+    if self.request_uri is not None and len(self.request_uri) < len(self.path):
```

Comment:
I'm a little puzzled by the second condition on lengths. Can you explain a scenario where this occurs?